src/envoy/http/jwt_auth/http_filter.cc

```diff
@@ -25,6 +25,14 @@
 
 namespace Envoy {
 namespace Http {
+namespace {
+// This function sanitizes the HTTP headers to remove the existing
+// JWT verification results specified by the key.
+void SanitizeJwtVerificationResultInHeaders(const LowerCaseString key,
```

Comment:
I felt not need to have this as a separate function, it only has one line, and only called in one place.

Reply:
I thought about that too. But I think using a function makes the code easier to understand and makes it easier to extend the sanitation code.

Reply:
+1 on remove the function. Have to scroll up to see what it does hurt readability too. Add function back when you really need to do more than just headers.remove()